### DISCUSSION TITLE
Is it possible to calculate an initial model by direct averaging of particles?

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#71] [Q&A] [answered]
Is it possible to calculate an initial model by direct averaging of particles? For example, the average of evenly spaced surface patches on virus particles?

**Comment by tofushan** [comment, marked as answer]
Yes, you could turn off the refinement in a Particle Refinement block, so no alignment will be performed. Select `Skip refinement` in the Refinement tab and unselect all the refinements under the Constrained refinement tab. 

If you do the pre-processing in nextPYP (same as the HIV-1 Gag tutorial), you should have initial particle orientations. That means that you should see the membrane in your average.

**Comment by JuhaHuiskonen** [comment]
Thanks! This makes sense.  One still has to give an initial template MRC file, but as alignments are skipped, this file can be a dummy file I suppose?

**Reply by tofushan** [reply]
Yes, correct.
